HelpSource/Classes/LFTri.schelp

```diff
@@ -14,7 +14,7 @@ classmethods::
 method::ar, kr
 
 argument::freq
-Frequency in Hertz.
+Frequency in Hertz. For efficiency reasons, only positive values yield in correct behavior.
 
```

Comment:
I don't think that "yield in sth." is correct. Better is "result in sth." here (or otherwise "yield sth.").